api/controllers/console/apikey.py

```diff
@@ -4,11 +4,13 @@
 from werkzeug.exceptions import Forbidden
 
 from extensions.ext_database import db
+from extensions.ext_redis import cache
 from libs.helper import TimestampField
 from libs.login import login_required
 from models.dataset import Dataset
 from models.model import ApiToken, App
 
+from ..service_api.wraps import get_api_token_from_db
```

Comment:
When it exceeds two levels, this is not our recommended practice. Perhaps using an absolute path would be better?

Reply:
fixit